scripts/metrics/ctest_metrics_xml_to_csv.py

```diff
@@ -0,0 +1,186 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+
+Scrapes metrics from CTest xml files and creates csv formatted files.
+"""
+import os.path
+import sys
+import csv
+import argparse
+import xml.etree.ElementTree as xmlElementTree
+import datetime
+
+from common import logging
+
+DEFAULT_CTEST_LOG_FILENAME = 'Test.xml'
+TAG_FILE = 'TAG'
+TESTING_DIR = 'Testing'
+
+def now_wrap():
+    """
+    Wrapper function for now() for unit testing purposes.
```

Comment:
Why was this necessary? Can you not directly mock datetime.datetime.now in the tests?

Reply:
Correct, it is not mockable

Reply:
https://docs.python.org/3/library/unittest.mock-examples.html#partial-mocking

Reply:
Fixed